taxonomy-editor-frontend/src/pages/editentry/ListAllEntryProperties.jsx

```diff
@@ -0,0 +1,21 @@
+import { Typography } from "@mui/material";
+import ListAllProperties from "./ListAllProperties";
+import ListTranslations from "./ListTranslations";
+
+// Parent component used for rendering sub-components
+// Sub-components are used to render info for an "entry"
+
+const ListAllEntryProperties = ({ nodeObject, setNodeObject }) => {
```

Comment:
could this component be integrated in the `AccumulateAllComponents`, this one is doing minimal stuff, but having it here you need to go 1 level down just to pass the state